src/db.py

```diff
@@ -125,6 +126,31 @@ def add_user(self, user: dict):
             )
             return STATUS_FAIL
 
+    def deactivate_user(self, user: dict):
+        session = self._sm()
+        db_user = session.query(User).filter(User.tg_id == user["tg_id"])
+        if db_user.count() == 0:
+            _logger.error(
+                f"User '{user['first_name']} {user['patronymic_name']} {user['last_name']}' with "
+                f"telegram id '{user['tg_id']}' does not exists in database. STATUS_FAIL"
+            )
+            return STATUS_FAIL
+
+        user = db_user.one()
+        user.status = "deactive"
+        session.commit()
+        status = user.status
+        session.close()
+        if status == "deactive":
+            _logger.info(f"Status of user with id '{user.id}' has been changed to 'deactive'. STATUS_OK")
+            return STATUS_OK
+        elif status == "activate":
+            _logger.warning(
+                f"Status of user with id '{user.id}' is already 'deactive'. "
```

Comment:
The message is confusing. We tried to change status, but it's still "active". (not "already deactive" )